openid-4-verifiable-presentations-1_0.md

```diff
@@ -1492,7 +1492,139 @@ issuers in Self-Sovereign Identity ecosystems using TRAIN</title>
+
+### Unsigned Request {#unsigned_request}
+
+The Verifier MAY send all the OpenID4VP request data as JSON elements in the `request` API parameter and receives the result in the API's `result` parameter. In this case, the Wallet will use the Verifier origin as asserted by the Browser as the Verifer's Client Identifier.
+
+The unsigned Request Object MUST NOT contain `client_id` and `client_id_scheme` parameters.
+
+### Signed Request {#signed_request}
+
+The Verifier MAY send a signed request, for example, when identification and authentication of the Verifier is required.
+
+The signed Request Object MAY contain all the parameters listed in (#browser_api_request), except `request`. The signed Request Object MUST contain an `expected_origins` parameter. 
+
+Below is a non-normative example of such a request:
+
+```js
+const credential = await navigator.identity.get({
+  digital: {
+    providers: [{
+      protocol: "urn:openid.net:oid4vp",
+      request: {
+        request: "eyJhbGciOiJF..."
+     }
+    }]
+  }
+});
+
+This is an example signed request payload:
+
+<{{examples/digital_credentials_api/signed_request_payload.json}}
+
+The signed request allows the Wallet to authenticate the Verifier using a trust framework other than the Web PKI utilized by the browser. An example of such a trust framework is the Verifier (RP) management infrastructure set up in the context of the eIDAS regulation in the European Union, in which case, the Wallet can no longer rely only on the web origin of the Verifier. This web origin MAY still be used to further strengthen the security of the flow. The external trust framework could, for example, map the Client Identifier to registered web origins.
+
+## Response
+
+Every OpenID4VP Authorization Request MUST result in a response being provided through the W3C Digital Credentials API. The Authorization Response is a JavaScript object, where the response parameters as defined for the Response Type are encoded as top-level members in this JavaScript object. 
+
+The following is a non-normative example of an OpenID4VP response that could be received from the W3C Digital Credentials API:
+	
+```js
+{
+  vp_token: "...",
+  presentation_submission: {...}
+}
+```
+
+Note: All mechanisms for cryptographically protecting the OpenID4VP response MAY be utilized with the W3C Digital Credentials API, too.
+
+# Examples with Credentials in Various Formats {#alternative_credential_formats}
 
 OpenID for Verifiable Presentations is Credential format agnostic, i.e., it is designed to allow applications to request and receive Verifiable Presentations and Verifiable Credentials in any Credential format. This section defines Credential Format Profiles for some of the known Credential formats. Other specifications or deployments can define their own Credential Format Profiles.
```

Comment:
I think this might be somewhat misleading, as although OID4VP may support any format, the wallets, browsers, OSs, etc. might not, and some could get blocked for security reasons (or they are just not supported).

Reply:
umm once the wallets, browsers, OSs, etc. start blocking credential formats, we can add a warning, but right now openid4vp supports any credential format.

Reply:
Right, I think we should maybe say that here. Something like:

> However, it is important to note that while OID4VP may support any format, compatibility with wallets, browsers, operating systems, etc., may vary. Some credential formats might not be supported or could be restricted by the wallets, browsers, operating systems for security, privacy, or compatibility reasons.